### ISSUE TITLE
Need new line inserted when <details> tag opened.

### ISSUE THREAD

**Issue by moraghughson** [rstacruz/details-polyfill#17]
This project is great - have had some users on IE or pre-Chromium-Edge that have benefitted from this. One thing that is not perfect that I thought you might like to know about is that it would be better to have a new line inserted when the `<detail>` tag is opened. Currently in IE, the content of the `<detail>` tag is on the same line as the opened tag. When you have nested tags this looks a bit odd.

I have tried changing the inserted style from `display:inline-block` to `display:block`, which fixes the newline problem but then the arrow is not on the same line as the rest of the `<detail>` tag so it's better in one way and worse in another. It could be because IE is not good at `display:inline-block`, but I thought you'd like to know anyway, in case you can make it better.

**Comment by moraghughson** [comment]
One small improvement I noticed after opening this issue is that if you add a style to details with `display: block`, it improves the first level of nesting. So something like this in the `injectStyle` - line 2 is the new line :-
```
injectStyle('details-polyfill-style',
  'html.no-details ' + DETAILS + ' { display: block; }\n' +
  'html.no-details ' + DETAILS + ':not([open]) > :not(' + SUMMARY + ') { display: none; }\n' +
  'html.no-details ' + DETAILS + ' > ' + SUMMARY + ':before { content: "\u25b6"; display: inline-block; font-size: .8em; width: 1.5em; }\n' +
  'html.no-details ' + DETAILS + '[open] > ' + SUMMARY + ':before { content: "\u25bc"; }' )
```

**Comment by rstacruz** [comment]
Thanks for the report! I'm not sure how to address this either but a PR would be appreciated.

There are other details polyfills out there to also try in case this one doesn't fit your use case.

**Comment by moraghughson** [comment]
I will look into making a Pull Request (PR). I am a new Git Hub user so bear with me!

**Comment by moraghughson** [comment]
Ok, I think I have done what you meant. To create a Pull Request, I first had to make the change to index.js in a clone and then do a Pull Request on that. I cannot for the life of me work out how to link the Pull Request to this Issue though. Help docs suggests I can do so, but the button it suggests I click just doesn't exist. Perhaps it is something that must be done by someone with write access to this repository, which I do not have. Do let me know if I have not done it right. First time in :-)

**Comment by moraghughson** [comment]
Hi @rstacruz  - I see this is still open. Is that because I haven't put it into the correct state? As I said, I am a new user on GitHub, so I may have missed some step in the process that comes naturally to you. In case it is not clear, I have made the change required and am ready for this to be merged in. Can you let me know if this branch is not in the correct state to merge in. Thanks.

**Comment by rstacruz** [comment]
I just merged it - sorry been a bit busy lately. I've published your changes as 1.2.0. :)

**Comment by moraghughson** [comment]
Thank you. I was concerned that I hadn't done something. Thank you so much for your help. Great to learn how all this process stuff works. Cheers!
